code/src/components/TodoItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+
+import { Button } from "lib/Button";
+import { CustomCheckbox } from "./CustomCheckbox";
+import { todos } from "../reducers/todos";
+import styled from "styled-components/macro";
```

Comment:
You've done a great job separating the outer and local dependencies! However I  think the import of styled components here can be moved to line nr 3 instead like you did in Addtodo.js? :)